### PR TITLE
BIG-242 region from genericbackendcredentials

### PR DESCRIPTION
**Pull request by martinjandl**
Jira: [CT-242](https://keboola.atlassian.net/browse/BIG-242)
Connection PR: [CT-242](https://github.com/keboola/connection/pull/4920)
SAPI PR: XXX
https://github.com/keboola/php-storage-driver-bigquery/pull/127

---

[CT-242]: https://keboola.atlassian.net/browse/CT-242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CT-242]: https://keboola.atlassian.net/browse/CT-242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by romanbracinik — APPROVED** [review]
🙆

**Review by romanbracinik — COMMENTED** [review]
hmm este premyslam ci to nedat len depricated https://github.com/keboola/storage-backend/pull/128/files

File: packages/php-storage-driver-common/proto/bucket.proto [hunk 1]
```diff
@@ -25,6 +25,7 @@ message CreateBucketCommand {
   }
 
   message CreateBucketBigqueryMeta {
+    option deprecated = true;
```
Comment (romanbracinik) [inline packages/php-storage-driver-common/proto/bucket.proto:28]: Hm pride mi, ze chceme deprekovat len ten region, ale takto deprekujes celu tu metu, https://github.com/keboola/storage-backend/pull/138/files#diff-cba0b541d7c54768da361b8094bdff7a67c1b80b98ecc45151f4e2e4b1e87efdR12 mozno sa mylim ale mozno casom na nieco budeme potrebova tu metu co na to @keboola/connection

**Review by romanbracinik — COMMENTED** [review]
este som si neni isty ohladne toho ci nedeprekovat len ten parameter

File: packages/php-storage-driver-common/proto/bucket.proto (hunk 1 above)
Comment (romanbracinik) [reply packages/php-storage-driver-common/proto/bucket.proto:28]: Este to prosim uprav https://keboolaglobal.slack.com/archives/C0553EM2RGX/p1711550127180529?thread_ts=1711524086.972039&cid=C0553EM2RGX

Comment (martinjandl) [reply packages/php-storage-driver-common/proto/bucket.proto]: ok, upravil jsem

**Review by romanbracinik — APPROVED** [review]
Lgtm 👌

**Pull request merged**